### PR TITLE
Correct Atlassian icon

### PR DESCRIPTION
**Pull request by mondeja**
atlassian

**Issue:** #3169 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

I've updated the logo using [this](https://www.atlassian.com/dam/jcr:242ae640-3d6a-472d-803d-45d8dcc2a8d2/Atlassian-horizontal-blue-rgb.svg) as source because Atlassian has updated their official logos in their [press kit](https://www.atlassian.com/company/news/press-kit).

### CODE REVIEW

**Comment by service-paradis** [comment]
@mondeja 
Looks different from the original. Maybe it has been updated since:
image

Also, it looks like the source in `simple-icons.json` is no more valid. Should probably be updated to https://www.atlassian.com/fr/company/news/press-kit

File: .svglint-ignored.json [hunk 1]
```diff
@@ -10,8 +10,6 @@
...
-    "M 12,0 A 12.007568,12.007568 0 0 0 0,12 12.034059,12.034059 0 0 0 12,24.030275 12.034059,12.034059 0 0 0 24,12 12.007568,12.007568 0 0 0 12.003784,0 Z m 0,0.90444656 A 11.09177,11.09177 0 0 1 23.099338,11.996216 c 0,6.130558 -4.964995,11.125828 -11.099338,11.125828 A 11.118259,11.118259 0 0 1 0.90823086,12 11.09177,11.09177 0 0 1 11.996216,0.90823086 Z M 12.08704,2.0662252 11.65563,3.3188269 5.9754032,19.402081 5.7710512,20.007569 H 18.440871 L 18.210029,19.402081 12.526017,3.3150427 Z m 0,2.8003785 2.009461,5.6499533 -3.027437,8.563859 H 7.050142 Z m 2.505204,7.0463573 2.535478,7.163671 h -5.097446 z": "Audio-Technica",
```
Comment (service-paradis) [inline .svglint-ignored.json]: Should not be removed from this PR :)

**Comment by mondeja** [comment]
I've updated the logo using [this SVG](https://www.atlassian.com/dam/jcr:242ae640-3d6a-472d-803d-45d8dcc2a8d2/Atlassian-horizontal-blue-rgb.svg) as source.

**Review by service-paradis — APPROVED** [review]

File: .svglint-ignored.json (hunk 1 above)
Comment (mondeja) [reply .svglint-ignored.json]: After merged, doesn't appears as removed from this pull. It's a very strange behaviour of GIT.

**Comment by mondeja** [comment]
Hey @service-paradis. Please, use "Squash and merge" option merging pulls :eyes:

**Comment by service-paradis** [comment]
> Hey @service-paradis. Please, use "Squash and merge" option merging pulls 👀

Oh damn!! You're right, sorry!

**Pull request merged**